tests/foreman/cli/test_oscap.py

```diff
@@ -673,10 +680,48 @@ def test_positive_associate_scap_policy_with_hostgroups(self):
         })
         self.assertEqual(scap_policy['hostgroups'][0], hostgroup['name'])
 
+    @run_only_on('sat')
+    @tier2
+    def test_positive_associate_scap_policy_with_hostgroups_via_ansible(self):
+        """Associate hostgroups to scap policy via ansible
+
+        :id: 2df303c6-bff5-4977-a865-a3afabfb8726
+
+        :setup:
+
+            1. Oscap should be enabled.
+            2. Oscap-cli hammer plugin installed.
+            3. More than 1 hostgroups
+            4. Ansible role and Ansible variable
+
+        :steps:
+
+            1. Login to hammer shell.
+            2. Execute "policy" command with "create" as sub-command.
+            3. Pass valid parameters and deploy option as ansible
+            4. Associate multiple hostgroups with policy
```

Comment:
I'm not sure if I understand correctly, are you apply multiple host groups here?  I see only one host group, not a list. please correct me if I'm wrong